# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/libgnomeui/pc-file-need-gnome-keyring.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./libgnomeui/libgnomeui-2.0.pc.in.orig	2004-12-04 21:11:03.097157568 +0100
+++ ./libgnomeui/libgnomeui-2.0.pc.in	2004-12-04 21:11:17.713935480 +0100
@@ -6,7 +6,7 @@
 
 Name: libgnomeui
 Description: libgnomeui
-Requires: libgnome-2.0 libgnomecanvas-2.0 gtk+-2.0 gdk-pixbuf-2.0 libart-2.0 gconf-2.0 libbonoboui-2.0
+Requires: libgnome-2.0 libgnomecanvas-2.0 gtk+-2.0 gdk-pixbuf-2.0 libart-2.0 gconf-2.0 libbonoboui-2.0 gnome-keyring-1
 Version: @VERSION@
 Libs: -L${libdir} -lgnomeui-2 @SM_LIBS@
 Cflags: -I${includedir}/libgnomeui-2.0
